### ISSUE TITLE
Received HTTP 202 Accepted instead of HTTP 500 when contract fails to deploy with an error

### ISSUE THREAD

**Issue by itamararjuan** [orbs-network/orbs-network-go#1433]
When deploying a contract which exceeds the maximum allowed compilation time, the web server returns an HTTP 202 Accepted due to timeout considerations (as it seems) instead of returning of returning an HTTP 500 due to the contract compilation timeout

This can be clearly seen by the logs on the one of the nodes when such deployment error occurs, attached is a portion of such log:

```
 {"level":"info","timestamp":"2019-11-05T14:05:03.593511377Z","message":"Sdk.Service.CallMethod failed","function":"virtualmachine.(*service).handleSdkServiceCallMethod","source":"/src/services/           virtualmachine/sdk_service.go:67","vcid":42,"node":"a328846cd5b4979d68a8c58a9bdfeee657b34de7","service":"virtual-machine","error":"compilation of deployable contract 'Experimental1572962688632594733'     failed: could not build a shared object: error building go source: signal: killed, go build output: ","callee":"Experimental1572962688632594733"}
 {"level":"info","timestamp":"2019-11-05T14:05:03.593588211Z","message":"contract returned error","function":"native.(*service).ProcessCall","source":"/src/services/processor/native/service.go:146",       "vcid":42,"node":"a328846cd5b4979d68a8c58a9bdfeee657b34de7","service":"processor-native","entry-point":"LeanHelix.Run","request-id":"LeanHelix.Run-a328846cd5b4979d68a8c58a9bdfeee657b34de7-                1572962592854828843","contract":"_Deployments","method":"deployService","error":"compilation of deployable contract 'Experimental1572962688632594733' failed: could not build a shared object: error        building go source: signal: killed, go build output: "}
 {"level":"info","timestamp":"2019-11-05T14:05:03.593672474Z","message":"transaction execution failed","function":"virtualmachine.(*service).runMethod","source":"/src/services/virtualmachine/execution.go: 66","vcid":42,"node":"a328846cd5b4979d68a8c58a9bdfeee657b34de7","service":"virtual-machine","result":"EXECUTION_RESULT_ERROR_SMART_CONTRACT","error":"compilation of deployable contract                    'Experimental1572962688632594733' failed: could not build a shared object: error building go source: signal: killed, go build output: ","transaction-or-query":"{ProtocolVersion:1,VirtualChainId:2a,       Timestamp:15d4493b3d34a006,Signer:{Scheme:(Eddsa){NetworkType:NETWORK_TYPE_TEST_NET,SignerPublicKey:07492c6612f78a47d7b6a18a17792a01917dec7497bdac1a35c477fbccc3303b,},},ContractName:_Deployments,         MethodName:deployService,InputArgumentArray:                                                                                                                                                                27000000020000001f0000004578706572696d656e74616c3135373239363236383836333235393437333300080000000000000001000000fd01000003000000f50100007061636b616765206d61696e0a0a696d706f727420280a09226769746875622e636 f6d2f6f7262732d6e6574776f726b2f636f6e74726163742d65787465726e616c2d6c69627261726965732d676f2f76312f6c697374220a09226769746875622e636f6d2f6f7262732d6e6574776f726b2f6f7262732d636f6e74726163742d73646b2f676f 2f73646b2f7631220a290a0a766172205055424c4943203d2073646b2e4578706f7274286765742c20616464290a7661722053595354454d203d2073646b2e4578706f7274285f696e6974290a0a66756e63205f696e69742829207b0a0a7d0a0a66756e632 067657428696e6465782075696e7436342920737472696e67207b0a096c697374203a3d206765744c69737428290a0972657475726e206c6973742e47657428696e646578292e28737472696e67290a7d0a0a66756e6320616464286974656d20737472696e 67292075696e743634207b0a0972657475726e206765744c69737428292e417070656e64286974656d290a7d0a0a66756e63206765744c6973742829206c6973742e4c697374207b0a0972657475726e206c6973742e4e6577417070656e644f6e6c794c697 37428226974656d73222c206c6973742e537472696e6753657269616c697a65722c206c6973742e537472696e67446573657269616c697a6572290a7d0a,}"}
```
The build output log for all nodes are available in the following CircleCI build link:
[Link to build](https://app.circleci.com/jobs/github/orbs-network/orbs-network-go/21688/tests)

**Expected behavior**

We expect such a timeout to propogate as an error from the native compiler so that the http server will handle it correctly and represent this error with a 500 HTTP error instead of a 202.

**Comment by netoneko** [comment]
Related to #1387

**Comment by electricmonk** [comment]
Also, we assume that the deployment failed due to timeout but the error doesn't mention a timeout. It *does* include an empty build output. When solving this issue, please make sure that if build failed due to timeout, the error indicates this as the cause.

**Comment by amir-arad** [comment]
should be fixed by #1434
